docs/setup/README.md

```diff
@@ -1,16 +1,16 @@
-# Verseagility Setup
-This part of the documentation helps you to set up Verseagility in our own subscription and with your data. Each of the points below will lead you to the respective documentation page. At the bottom of each page you will find a link to the previous and next page.
+# Getting Started with Verseagility
+This part of the documentation helps you to set up Verseagility in our own subscription and with your data, step by step. Each of the points below will lead you to the respective documentation page. At the bottom of each page you will find a link to the previous and next page.
 
-1. [Setup](01%20-%20Verseagility%20Setup.md)
+1. [Setup of Code and Infrastructure](01%20-%20Verseagility%20Setup.md)
```

Comment:
Should we also rename the Title of the linked pages to be a little more consistent?